pyk/pyproject.toml

```diff
@@ -6,21 +6,23 @@ build-backend = "poetry.core.masonry.api"
 name = "pyk"
 version = "0.1.0"
 description = ""
-authors = ["Runtime Verification, Inc."]
+authors = [
+    "Runtime Verification, Inc. <contact@runtimeverification.com>",
+]
 
 [tool.poetry.dependencies]
 python = "^3.8"
 graphviz = "^0.20.1"
 
 [tool.poetry.dev-dependencies]
-autoflake = "^1.4"
-black = "^22.6.0"
-flake8 = "^5.0.4"
-flake8-bugbear = "^22.7.1"
-flake8-comprehensions = "^3.10.0"
-isort = "^5.10.1"
-mypy = "^0.971"
-pep8-naming = "^0.13.1"
+autoflake = "*"
+black = "*"
+flake8 = "*"
+flake8-bugbear = "*"
+flake8-comprehensions = "*"
+isort = "*"
+mypy = "*"
+pep8-naming = "*"
```

Comment:
This will pull in the latest version of each?

Reply:
This expresses that any version of the respective dependency is accepted. So when we run `poetry install` for the first time, Poetry will not have any additional constraints other than what these dependencies may put on each other's version. In practice, Poetry will probably favor the most recent versions unless there is a conflict.

Note that we still have full control on the version pf these dependencies, as `poetry.lock`, committed to the repo, freezes them to a given version, and the we can decide when to bump a given tool to a newer version (like for example in e6b08046f5709ab8b66fc950943f677324861383). It is just not managed on the template level.

Reply:
Sounds good.